src/components/FileTree/index.spec.tsx

```diff
@@ -496,5 +500,58 @@ describe(__filename, () => {
       const isNodeExpanded = treeFold.prop('isNodeExpanded');
       expect(isNodeExpanded(node)).toBeFalsy();
     });
+
+    it('logs a warning message when no version is loaded', () => {
+      const _log = createFakeLogger();
+
+      render({
+        _log,
+        version: createInternalVersion({ ...fakeVersion, id: 0 }),
+      });
+
+      expect(_log.warn).toHaveBeenCalled();
+    });
+
+    it('renders a Loading component when no version is loaded', () => {
+      const root = render({
+        version: createInternalVersion({ ...fakeVersion, id: 0 }),
```

Comment:
The `id: 0` bit is a little confusing. Could you use `versionId: 1234` in these other tests, too?

Reply:
Yes, I cleaned up the `render` function to accept just a `versionId` and it's much better now.